tests/release/pipelines/push_to_external_registry.go

```diff
@@ -172,10 +172,11 @@ var _ = framework.ReleasePipelinesSuiteDescribe("Push to external registry", Lab
 			component, err := fw.AsKubeAdmin.HasController.GetComponent(component.GetName(), devNamespace)
 			Expect(err).ShouldNot(HaveOccurred(), fmt.Sprintf("could not get component %s in the %s namespace", component.GetName(), devNamespace))
 			containerImageDigest := strings.Split(component.Spec.ContainerImage, "@")[1]
+			imageDigest := strings.Replace(containerImageDigest, ":", "-", -1)
 
-			imageDigest, err := releasecommon.GetDigestWithTagInQuay(releasecommon.ReleasedImagePushRepo + ":latest")
-			Expect(err).ShouldNot(HaveOccurred(), fmt.Sprintf("failed while getting Digest for quay image %s with error: %+v", releasecommon.ReleasedImagePushRepo + ":latest", err))
-			Expect(imageDigest).To(Equal(containerImageDigest))
+			tagExist, err := releasecommon.DoesTagExistsInQuay(releasecommon.ReleasedImagePushRepo + ":" + imageDigest)
```

Comment:
are you really looking for a tag? you just want to see if the digest is there.

Reply:
Changed to check digest. Thanks